### PR TITLE
Switch to masses instead of atom ints

### PR DESCRIPTION
**Pull request by charnley**

### CODE REVIEW

File: ppqm/chembridge.py [hunk 1]
```diff
@@ -450,29 +450,55 @@ def get_canonical_smiles(smiles: str) -> str:
...
+        atomic_masses (np.ndarray): 
+            numpy array of atomic masses. Must have shape (n_atoms)
+        coordinates (np.ndarray):
+            numpy array of coordinates. Must have shape (n_atoms, 3)
+
+    returns:
+        The center of mass of the molecule, as a numpy array of shape (3)
+    """
 
+    assert len(atomic_masses.shape) == 1, "Input array has wrong shape"
```
Comment (charnley) [inline ppqm/chembridge.py]: We shouldn't use assert in function, but raise exceptions. Otherwise it is harder for others to use the code

Comment (hagenmuenkler) [reply ppqm/chembridge.py]: addressed in commit de1757a.

File: tests/test_chembridge.py [hunk 2]
```diff
@@ -208,6 +208,9 @@ def test_get_dipole_moments() -> None:
     molobj = tasks.generate_conformers(molobj, n_conformers=n_conformers)
 
     moments = chembridge.get_dipole_moments(molobj)
+
+    print(moments)
```
Comment (hagenmuenkler) [inline tests/test_chembridge.py:212]: is the print statement still needed here?

**Review by hagenmuenkler — APPROVED** [review]
looks good to me now

File: tests/test_chembridge.py (hunk 2 above)
Comment (charnley) [reply tests/test_chembridge.py:212]: Print statements in tests are fine, as they only get printed out on test failures

**Pull request merged**